### DISCUSSION TITLE
nest.js + onetable module

### DISCUSSION THREAD

**Discussion by onhate** [sensedeep/dynamodb-onetable#495] [Show and tell]
https://github.com/onhate/nest-onetable

Just sharing the Nest-Onetable library I just release.

This library provides a convenient way to integrate the power of [DynamoDB OneTable](https://github.com/sensedeep/dynamodb-onetable) with [NestJS](https://nestjs.com/) applications. It simplifies the process of creating and working with DynamoDB tables using the OneTable library within your NestJS modules.

I've been using this in production already and now I'm sharing it.

Feel free to comment :)

**Comment by mobsense** [comment]
That is AWESOME. Well done.
